src/FGraphDiff/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# locate the necessary dependencies, if any
+
+project(FGraphDiff)
+
+# extra header files
+SET(headers
+    ../FGraph/mrob/node.hpp
```

Comment:
you should not include node this way, it is included when you link libraries. Remove.